### ISSUE TITLE
Translate phone types to be acceptance to SUDS

### ISSUE THREAD

**Issue by colinpmacarthur** [18F/fs-middlelayer-api#67]
SUDS requires a certain set of phone types that are sometimes non-sensical. The middle layer should translate from sensical phone types to SUDS' ones.

#Tasks:
- [X] change schema so that all phone number types are just business

**Comment by lauraGgit** [comment]
Part of [FFP #76](https://github.com/18F/fs-permit-platform/issues/76)

**Comment by tadhg-ohiggins** [comment]
Do we have the list of phone types we should be translating to?

**Comment by lauraGgit** [comment]
Here's what I learned: "Phone number types in DB are FAX, BUSINESS, BUSINESS 2. I think the first two should be ok."

- So lets just have it be default to `BUSINESS`

**Comment by lauraGgit** [comment]
🕴

**Comment by lauraGgit** [comment]
Waiting on #153 being merged into dev

**Comment by lauraGgit** [comment]
Closed by #112
